open_event/views/admin/models_views/events.py

```diff
@@ -0,0 +1,59 @@
+from flask import request, url_for, redirect
+from flask_admin import expose
+from flask_admin.contrib.sqla import ModelView
+from flask.ext import login
+from ....helpers.data import DataManager
+from ....helpers.data_getter import DataGetter
+
+
+class EventsView(ModelView):
+
+    def is_accessible(self):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/triplez/open-event-orga-server/file/2673041373/issues/source?bid=3233864&fileBranchId=3324174#l11)